app/src/main/kotlin/com/wire/android/ui/home/conversations/model/MessageTypes.kt

```diff
@@ -131,10 +135,9 @@ fun MessageButtonsContent(
         for (index in buttonList.indices) {
             val button = buttonList[index]
             MessageButtonItem(
-                messageId = messageId,
-                pendingButtonId = pendingButton,
+                pendingButtonId = viewModel.pendingButtonId,
                 button = button,
-                onClick = onClick
+                onClick = viewModel::onButtonClicked
```

Comment:
we could use here Clickable it has a enabled variable that would work here I think 

Clickable(enabled = true)

Reply:
if we do that all above, MessageButtonItem, will only need the text to be passed and not the whole MessageButton it will make the Composable abnit more usable, not sure if that makes any sense out of the context of the MessageButton, but it wont hurt I think

Reply:
the button state changes depending on different variables, I will rename the onClick to doButtonAction so it is clear what this lambda is about, if i pass a clickable(true, onclick: () -> Unit) then it contradict with d disabling the button on line 170

Reply:
I could be reading the code wrong too, if that is the case then I am sorry but from what I understand if the button is not selected we want to disable clickiing on it, so this should do the trick

Reply:
and makes the code look a bit easier in my opinion, because of those remember stuff being removed